sdk/servicebus/service-bus/.scripts/buildBrowserTestResources.js

```diff
@@ -1,319 +0,0 @@
-const msRestNodeAuth = require("@azure/ms-rest-nodeauth");
```

Comment:
Browser CI is failing since this file is removed. Do we have another way to generate resources for browser testing?

Reply:
Oh I thought @ramya-rao-a had updated the package.json script to be the right one.. updating it.
The script is no longer used since we'll be now using the atom based management API for creating resources in browser

Reply:
Opened https://github.com/Azure/azure-sdk-for-js/pull/6908 once that is in, can rerun the tests